backend/src/main/java/com/example/backend/BackendConfigServlet.java

```diff
@@ -27,13 +27,13 @@
 import com.google.gson.Gson;
 import com.google.protobuf.FieldMask;
 import com.google.type.LatLng;
-import google.maps.fleetengine.delivery.v1.CreateDeliveryVehicleRequest;
-import google.maps.fleetengine.delivery.v1.CreateTaskRequest;
-import google.maps.fleetengine.delivery.v1.DeliveryServiceGrpc;
-import google.maps.fleetengine.delivery.v1.DeliveryVehicle;
-import google.maps.fleetengine.delivery.v1.DeliveryVehicleLocation;
-import google.maps.fleetengine.delivery.v1.Task;
-import google.maps.fleetengine.delivery.v1.UpdateDeliveryVehicleRequest;
+import com.google.maps.fleetengine.delivery.v1.CreateDeliveryVehicleRequest;
```

Comment:
I have changes in Piper that keep the imports in the right order. Let's figure out how to get that merged in.